### ISSUE TITLE
Registering a 1.6 package fails due to LazyArtifacts

### ISSUE THREAD

**Issue by maleadt** [JuliaRegistries/Registrator.jl#318]
https://github.com/JuliaGPU/CUDA.jl/commit/bfb5d734566e0758493716bec78f6616d64d745c#commitcomment-45769140

> Error while trying to register: Error in (Julia)Project.toml: Package 'LazyArtifacts' with UUID: 4af54fe1-eca0-43a8-85a7-787d91b784e3 not found in registry or stdlib

**Comment by StefanKarpinski** [comment]
cc @staticfloat

**Comment by DilumAluthge** [comment]
@maleadt As a workaround for now, you could use LocalRegistry.jl to make the registration. If you run LocalRegistry with Julia 1.6 or 1.7, it should work.

**Comment by staticfloat** [comment]
I don't really know what to do here; I guess whatever bot is running the registration needs to be running Julia 1.6?

**Comment by DilumAluthge** [comment]
Yeah. @nkottary any chance we can deploy Registrator on the 1.6 beta?

**Comment by DilumAluthge** [comment]
@maleadt See this PR, which registers CUDA v2.5.0 via LocalRegistry.jl: https://github.com/JuliaRegistries/General/pull/27666
